sdk/servicebus/armservicebus/README.md

```diff
@@ -0,0 +1,76 @@
+# Azure Container Registry Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/servicebus/armservicebus)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/servicebus/armservicebus)
+
+The `armservicebus` module provides operations for working with Azure Container Registry.
+
+[Source code](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/servicebus/armservicebus)
+
+# Getting started
+
+## Prerequisites
+
+- an [Azure subscription](https://azure.microsoft.com/free/)
+- Go 1.13 or above
+
+## Install the package
+
+This project uses [Go modules](https://github.com/golang/go/wiki/Modules) for versioning and dependency management.
+
+Install the Azure Container Registry module:
+
+```sh
+go get github.com/Azure/azure-sdk-for-go/sdk/servicebus/armservicebus
+```
+
+## Authorization
+
+When creating a client, you will need to provide a credential for authenticating with Azure Container Registry.  The `azidentity` module provides facilities for various ways of authenticating with Azure including client/secret, certificate, managed identity, and more.
+
+```go
+cred, err := azidentity.NewDefaultAzureCredential(nil)
+```
+
+For more information on authentication, please see the documentation for `azidentity` at [pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity).
+
+## Connecting to Azure servicebus
+
+Once you have a credential, create a connection to the desired ARM endpoint.  The `armcore` module provides facilities for connecting with ARM endpoints including public and sovereign clouds as well as Azure Stack.
+
+```go
+con := armcore.NewDefaultConnection(cred, nil)
+```
+
+For more information on ARM connections, please see the documentation for `armcore` at [pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/armcore](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/armcore).
+
+## Clients
+
+Azure Container Registry modules consist of one or more clients.  A client groups a set of related APIs, providing access to its functionality within the specified subscription.  Create one or more clients to access the APIs you require using your `armcore.Connection`.
+
+```go
+client := armservicebus.NewRegistriesClient(con, "<subscription ID>")
```

Comment:
Please confirm we have a function `NewRegistriesClient` in this package. If not, please randomly choose one `NewXXXClient` function in this module and put it here.